grunttasks/rev.js

```diff
@@ -9,8 +9,7 @@ module.exports = function (grunt) {
         src: [
           '<%= yeoman.dist %>/bower_components/**/*.js',
           '!<%= yeoman.dist %>/bower_components/jquery-ui/**/*.js',
```

Comment:
jquery-ui does not actually use requireOnDemand and can probably be removed from this list.